examples/Learning_example/learning_example.ipynb

```diff
@@ -0,0 +1,2295 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import sys 
</pre><p>Could you add a markdown cell at the top explaining what this notebook does?</p><p> </p><p>Also need to mention that you need sklearn and torch in addition to regular paseos requirements.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>